amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc

```diff
@@ -0,0 +1,34 @@
+= Amazon SQS Connector 5.6 Additional Configuration Information - Mule 4
+:page-aliases: connectors::amazon/amazon-sqs-connector-config-topics.adoc
+
+[node-behavior]
+== Receiving Messages in Cluster Mode
+
+The default behavior of the Receive Messages input source is to receive messages on all nodes when running in cluster mode. To have the app receive messages on the primary node only, select the *Primary node only* checkbox on the *Advanced* sub-tab on the *Receive messages* tab.
+
+The multi-node cluster behavior depends on the selected SQS queue type. When using the Standard queue, SQS guarantees that a message is delivered at least once. However, duplicate messages can be introduced in the queue. You can use a FIFO queue to ensure that a message is delivered only once and that it remains available until a consumer processes and deletes it.
+
+For more information about queue types, see https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/standard-queues.html[AWS SQS Stanard Queues] and https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html[AWS SQS FIFO Queues].
+
+== Creating the FIFO Queue
+
```

Comment:
FIFO (First-In-First-Out) queues deliver messages in the order in which they are sent.

Reply:
Nice to add this